docs/releases/g_1_1.md

```diff
@@ -0,0 +1,17 @@
+# iRobot® Create® 3 Release G.1.1
+
+This is the first production Create® 3 release.
+
+## This release is running ROS 2 Galactic with the following interface library versions:
+
+- [irobot_create_msgs - 1.2.2](https://github.com/iRobotEducation/irobot_create_msgs/tree/1.2.2)
+- [cyclonedds - 0.8.1](https://github.com/eclipse-cyclonedds/cyclonedds/tree/0.8.1)
+- [Fast-DDS - 2.3.3](https://github.com/eProsima/Fast-DDS/tree/2.3.3)
+
+## Release Overview
+
+This release provides the robot's sensors and controls via topics, actions, and services, see [ROS 2 API](../../api/ros2/).
+
+The robot runs a webserver for updating software, configuring to the network, getting error logs from the robot, and configuring ROS 2 environment variables and parameters.
+This release has some known issues using the network configuration webserver page to associate with networks with non standard characters.
```

Comment:
Change "non standard characters" to "certain characters."

Reply:
Do we know all the problematic characters? If yes we should list them here